EmpireLegends/GameStart.cpp

```diff
@@ -0,0 +1,217 @@
+#include <iostream>
+#include <sstream>
+
+#include "MapLoader.h"
+#include "Player.h"
+#include "BiddingFacility.h"
+
+using namespace std;
+
+int main() {
```

Comment:
I don't think the game start should have a driver format. I'm pretty sure it should be a class on its own. And then we can have a driver testing its functionality. Because I'm not sure how we're supposed to integrate the result of this `main` into the main loop